config/metrics.js

```diff
@@ -113,6 +113,10 @@ module.exports = {
 		extract: (item) => !!getBPRules(item)?.allowsDeletions,
 		permissions: ['ADMIN'],
 	},
+	AllowsSecurity: {
+		extract: (item) => !!getBPRules(item)?.AllowsSecurity,
+		permissions: ['ADMIN'],
```

Comment:
the security policy can be set by anyone with write access, so i think this can support WRITE or above.